src/app/js/views/config.js

```diff
@@ -3,23 +3,19 @@ import PropTypes from 'prop-types'
 import i18n from '../utils/i18n.js'
 import classNames from 'classnames'
 
-export
-default class ConfigView extends Component {
-  static displayName = 'ConfigView';
-  static propTypes = {
-    config: PropTypes.object,
-    ipfs: PropTypes.object
-  };
-
-  state = {
-    body: JSON.stringify(this.props.config, null, 2),
-    error: null,
-    saving: false,
-    saved: false,
-    height: 0
-  };
+class ConfigView extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      body: JSON.stringify(this.props.config, null, 2),
+      error: null,
+      saving: false,
+      saved: false,
+      height: 0
+    }
+  }
 
-  updateHeight = () => {
+  updateHeight () {
```

Comment:
same here, you can't just change this syntax, it means that `this` will not be bound anymore